flask_appbuilder/security/manager.py

```diff
@@ -347,6 +348,10 @@ def auth_user_registration(self):
     def auth_user_registration_role(self):
         return self.appbuilder.get_app.config["AUTH_USER_REGISTRATION_ROLE"]
 
+    @property
+    def auth_user_registration_role_jmespath(self):
```

Comment:
nit: I'm starting to add type annotations, at least on every new PR.

`def auth_user_registration_role_jmespath(self) -> str:`